src/components/EditEventForm/index.jsx

```diff
@@ -249,6 +249,21 @@ const EditEventForm = props => {
             className="discard-button">
             Discard
           </Button>
+          <Button
+            type="danger"
+            onClick={() => {
+              props
+                .deleteEvent(props.values.uuid)
+                .then(resp => {
+                  history.push('/');
+                })
+                .catch(error => {
+                  console.log(error);
+                });
+            }}
```

Comment:
Can we pull this function out and pass it down as a prop from the container? Ideally we don't have too many anonymous functions inline within our /components because it overcrowds it.